libqtile/widget/notify.py

```diff
@@ -57,6 +58,13 @@ class Notify(base._TextBox):
         ),
         ("audiofile", None, "Audiofile played during notifications"),
         ("action", True, "Enable handling of default action upon right click"),
+        (
+            "parse_text", None, "Function to parse and modify notifications. "
+            "e.g. function in config that removes line returns:"
+            "def my_func(text)"
```

Comment:
Two things:
1) Missing a `:` after the definition
2) This formatting won't be preserved in our docs - it will just look like:

> Function to parse and modify notifications. e.g. function in config that removes line returns:def my_func(text) return text.replace(’n’, ‘’)then set option parse_text=my_func

Reply:
Other than that, code runs fine so I can merge once this is fixed. I'll do the test for it.

Reply:
I just copy pasted it from another widget so I'd say let's stay consistent for now and fix both in a later PR. :)

Reply:
Fine. I'll add to my list!